docker-compose.yml

```diff
@@ -113,6 +113,7 @@ services:
   # ----------------------------------------------------------------------
   rstudio:
     hostname: rstudio
+    platform: linux/amd64
```

Comment:
How sure are you this one's needed? I see ARM builds of this one on Dockerhub: https://hub.docker.com/r/pecan/rstudio-nginx/tags